examples/create-window/main.go

```diff
@@ -96,12 +96,22 @@ func main() {
 			fmt.Println("Both event and error are nil. Exiting...")
 			return
 		}
-
+		
```

Comment:
Please, always format go code with ```go fmt```. If you are using an IDE, configure it to run formatting after save.